src/components/ATeamAlert.js

```diff
@@ -16,7 +16,7 @@ export const AlertAteamApiDown = () => {
         if(accessToken) {
             testAteamAPI(accessToken, setStatus);
         }
-    }, [accessToken]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
@ianlongden I forget what this was for, but I think I prevented new warnings showing up on the browser console.  It's like a # noqa  which is needed.

Reply:
There are no warnings being generated wrt this so guessing some thing else was removed previously that needed this.

Reply:
Cool, sounds good